subgraphs/eip1155/mappings/index.ts

```diff
@@ -50,10 +51,8 @@ export function handleTransferBatch(event: TransferBatch): void {
     from.updatedAt = event.block.timestamp;
     from.save();
   }
-  from.totalTokens = event.params._from.equals(Address.zero())
-    ? from.totalTokens
-    : from.totalTokens.minus(BigInt.fromI32(1));
-  from.totalTransactions = from.totalTransactions.plus(BigInt.fromI32(1));
+  from.totalTokens = event.params._from.equals(Address.zero()) ? from.totalTokens : from.totalTokens.minus(ONE_BI);
```

Comment:
Use `ZERO_ADDRESS` for `Address.zero()`?

Reply:
Good catch, there was a few other places as well, updated in https://github.com/LooksRare/subgraph/pull/98/commits/29c216a05f4f81e3d35ada99aa641514884d1227